TenancyInformationApi/V1/Infrastructure/UhAgreementType.cs

```diff
@@ -0,0 +1,15 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.AspNetCore.Mvc;
+
+namespace TenancyInformationApi.V1.Infrastructure
+{
+    [Table("lookup", Schema = "dbo")]
+    public class UhAgreementType
+    {
+        // [Column("lu_type")] private const string LookupType = "ZAG";
+        [StringLength(1)]
+        [Column("lu_ref"), Key] public string UhAgreementTypeId { get; set; }
```

Comment:
Maybe change the type to `char` instead of a string of length 1.

Reply:
If i remember right, it *really* didn't like it when I used a `char`. Will have to have another look.

Reply:
No worries then, just a thought :)